_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -93,6 +93,14 @@ Not all of [visible][] label is included in [accessible name][].
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbols cannot be substituted for text as [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
+
+```html
+<a href="/" aria-label="Proof of two multiplied by two is four">Proof of 2&times;2=4</a>
```

Comment:
I like adding the math example, but aria-label should be “Proof of two times two equals four” and not “Proof of two multiplied by two is four”, as per the guidance in the Understanding page link.   (Also, a more literal aria-label might help with translation too.)